tests/testApproximate.py

```diff
@@ -53,7 +53,8 @@ def makeRamp(self, binsize=1):
         rampCoeffs = (1000, 1, 1)
         for i in range(ramp.getHeight()):
             for j in range(ramp.getWidth()):
-                ramp.set(j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
+                ramp.set(
+                    j, i, (rampCoeffs[0] + rampCoeffs[1]*x[j] + rampCoeffs[2]*y[i], 0x0, var))
```

Comment:
How about splitting before `(rampCoeffs[0]`? Keeping the tuple on one line seems like better style than splitting it in half.